src/components/Tabs/styles.scss

```
@@ -23,6 +24,37 @@
       color var(--sds-anim-transition-default),
       background-color var(--sds-anim-transition-default);
 
+    &--with-border {
+      &[data-is-active="true"] {
+        &:after {
+          content: "";
+          height: 2px;
+          width: 95%;
+          background: var(--sds-clr-lilac-11);
+          position: absolute;
+          bottom: -2px;
+          left: 0;
+          left: 50%;
+          transform: translateX(-50%);
+        }
+        background-color: transparent;
+        --Tabs-default-text: var(--sds-clr-lilac-11);
+        --Tabs-default-background: transparent;
+        border-radius: 0;
```

Nit: CSS variables should be as high up as possible to make it easier to notice them and not introduce unexpected behavior.

```suggestion
        --Tabs-default-text: var(--sds-clr-lilac-11);
        --Tabs-default-background: transparent;
        
        &:after {
          content: "";
          height: 2px;
          width: 95%;
          background: var(--sds-clr-lilac-11);
          position: absolute;
          bottom: -2px;
          left: 0;
          left: 50%;
          transform: translateX(-50%);
        }
        background-color: transparent;
        border-radius: 0;
```